code/game/objects/items/RPD.dm

```diff
@@ -557,21 +580,29 @@ GLOBAL_LIST_INIT(RPD_recipes, list(
 			playsound(get_turf(src), 'sound/machines/click.ogg', 50, 1)
 			if(do_after(user, 2, target = A))
 				activate()
-				var/obj/item/pipe/P = new (A, pipe_type=queued_p_type, dir=queued_p_dir)
+				var/obj/item/pipe/P = new (A, _pipe_type=queued_p_type, _dir=queued_p_dir)
```

Comment:
Named args dont work with initialize

Reply:
... I think

Reply:
Ya, as far i could figure out, there is no way to do that

Reply:
@Cyberboss i actually changed it to a diff named arg after the old one had errors
and when the names mismatched it actually started runtiming lol

Reply:
nvm yeah it doesnt work im trying to fix it atm